openclsim/core.py

```diff
@@ -932,7 +961,7 @@ def calc_depth_restrictions(self, location, processor):
                 seconds=processor.unloading(
                     self,
                     location,
-                    self.container.level - filling_degree * self.container.capacity,
+                    self.container.level + filling_degree * self.container.capacity,
```

Comment:
Really?

Reply:
I can not comment on this since I have not touched this part of the code. Who did the change?

Reply:
If I turn it to minus the test fails. If it is a plus, the test passed. Thus, I kept it as a plus!

Reply:
Haha, I did not change it. If we really want to know you can check with the GitLens extension in VS Code.